tox.ini

```diff
@@ -1,28 +1,35 @@
 [tox]
 envlist =
     py27,
+    py34,
+    py35,
```

Comment:
I think Python 3.4 and 3.5 do not have to be supported. Nobody who will port a Python project to Python 3 needing this library will run on these old versions. (The Plone developers also only support Python 3.6+)

Reply:
The current Debian stable (stretch) has only Python 3.5.

I don't know if many people deploy software on Debian stretch, but if they do, they might be interested in 3.5.

Reply:
I will only add support for Python > 3.5.